libbeat/scripts/generate_template.py

```diff
@@ -273,6 +273,12 @@ def fill_field_properties(args, field, defaults, path):
         target += "-es2x"
     target += ".json"
 
-    with open(args.path + "/etc/fields.yml", 'r') as input:
+    with open(args.path + "/etc/fields.yml", 'r') as f:
+        fields = f.read()
+
+        # Appends beat fields from libbeat
+        with file(args.path + "/../libbeat/_beat/fields.yml") as f:
```

Comment:
I suspect this might be unfriendly to the beat generator? Might want to introduce the concept of a beats path already.